scripts/compare_fitted_nuisance.py

```diff
@@ -0,0 +1,103 @@
+import ROOT
+import sys
+import json
+import pyhf
+import click
+
+@click.command()
+@click.option(
+    '--root-workspace',
+    help='The location of the root file containing the combined root workspace'
+)
+@click.option(
+    '--pyhf-json',
+    help='The location of the json file containing the pyhf likelihood info'
+)
+
+@click.option(
+    '--outfile',
+    help='Path to file to output nuisance parameter comparison to. Will print to screen if left blank.',
+    default=""
+)
+
+def compare_fitted_nuisance(root_workspace, pyhf_json, outfile):
+  # Get the root fit results 
+  infile = ROOT.TFile.Open(root_workspace)
+  workspace = infile.Get("combined")
+
+  mc = workspace.obj('ModelConfig')
+
+  def exhaust_argset(s):
+    it = s.fwdIterator()
+    pars = []
+    while True:
+      n = it.next()
+      if not n:
+        break
+      yield n
+
+  pars = [
+    x.GetName() for x in exhaust_argset(mc.GetNuisanceParameters())
+   ] + [
+    x.GetName() for x in exhaust_argset(mc.GetParametersOfInterest())]
+
+  p = workspace.pdf('simPdf')
+  d = workspace.data('obsData')
+  p.fitTo(d)
+
+  params_root = {}
+  for p in pars:
+    params_root[p] = workspace.var(p).getVal()
```

Comment:
```suggestion
  model = workspace.pdf('simPdf')
  data = workspace.data('obsData')
  model.fitTo(data)

  params_root = {}
  for par in pars:
    params_root[par] = workspace.var(par).getVal()
```

It would be nice if the variables names were more explicit here.